web/viewer-snippet-mozPrintCallback-polyfill.html

```diff
@@ -1,79 +1,13 @@
-<div id="mozPrintCallback-shim" hidden>
-  <style>
-@media print {
-  #printContainer div {
```

Comment:
I'd expect that removing this line (as part of the code move that I suggested against), can actually cause issues.